src/c++/perf_analyzer/test_profile_data_exporter.cc

```diff
@@ -32,31 +32,57 @@ namespace triton { namespace perfanalyzer {
 
 TEST_CASE("profile_data_exporter: ConvertToJson")
 {
+  using std::chrono::nanoseconds;
+  using std::chrono::system_clock;
+  using std::chrono::time_point;
+
   MockProfileDataExporter exporter{};
 
   InferenceLoadMode infer_mode{4, 0.0};
   uint64_t sequence_id{1};
 
-  auto clock_epoch{std::chrono::time_point<std::chrono::system_clock>()};
-  auto request_timestamp{clock_epoch + std::chrono::nanoseconds(1)};
-  auto response_timestamp1{clock_epoch + std::chrono::nanoseconds(2)};
-  auto response_timestamp2{clock_epoch + std::chrono::nanoseconds(3)};
-  std::vector<std::string> bufs{"abc", "def", "ghi", "jkl"};
+  auto clock_epoch{time_point<system_clock>()};
+  auto request_timestamp{clock_epoch + nanoseconds(1)};
+  auto response_timestamp1{clock_epoch + nanoseconds(2)};
+  auto response_timestamp2{clock_epoch + nanoseconds(3)};
+
+  // Request inputs
+  std::vector<std::string> in_bufs{"123", "456", "true"};
+  RequestRecord::RequestInput request_input{
+      {"in_key1",
+       {reinterpret_cast<const uint8_t*>(in_bufs[0].data()), in_bufs[0].size(),
+        "BYTES"}},
```

Comment:
I see you have bytes, int32, and bool. Let's also add JSON to this test?

Reply:
added JSON and missing test checks.